packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -129,7 +129,7 @@ const Select = <T extends OptionValueType>({
   }
 
   return (
-    <div className={twMerge('relative', className)} ref={ref}>
+    <div className={twMerge('relative w-full', className)} ref={ref}>
```

Comment:
@CITIZENDOT same here - think it will cause problems?

Reply:
Didn't encounter any issues with this.